src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import "./introduce.scss";
+
+const Self = () => {
+  const navigate = useNavigate();
+  return (
+    <div className="main">
```

Comment:
```suggestion
    <div className="introduce">
```
컴포넌트의 가장 최상위 부모는 컴포넌트의 이름과 동일하게 구현해주세요!
단, 소문자로 시작하는 형태로 구현해주세요~